wp-job-manager-functions.php

```diff
@@ -1293,13 +1293,14 @@ function job_manager_upload_file( $file, $args = array() ) {
 		return $file;
 	}
 
+	$allowed_file_extensions = implode( ', ', str_replace( '|', ', ', array_keys( $allowed_mime_types ) ) );
```

Comment:
Suggestion: Add a comment here to explain _why_ you are replacing the pipes.

Reply:
Done! 92e7af202162eddd7c565f11ad49c951cdd940cf

I also moved the new var into just the scope in which it is used.